onboardingtaskapp.client/src/components/Customer/Create.jsx

```diff
@@ -23,31 +29,45 @@ export default function Create() {
     const handleSubmit = async (e) => {
         e.preventDefault();
         try {
-            const response = await fetch('https://localhost:7207/api/Customer', {
+            const response = await fetch(`https://onboardsite.azurewebsites.net/api/Customer`, {
                 method: 'POST',
                 headers: {
                     'Content-Type': 'application/json'
                 },
                 body: JSON.stringify(formData)
             });
+            console.log(response)                       
```

Comment:
Can you remove this? We need to remove console.log in product level so users won't be able to see any unwated data. Thanks